kt/plugins/godot-intellij-plugin/src/main/kotlin/godot/intellij/plugin/module/GodotModuleBuilder.kt

```diff
@@ -114,36 +118,86 @@ class GodotModuleBuilder : ModuleBuilder(), ModuleBuilderListener {
+            copyTemplateFile(basePath, "gradle/wrapper/gradle-wrapper.jar")
+            copyTemplateFile(basePath, "gradle/wrapper/gradle-wrapper.properties")
+            copyTemplateFile(basePath, "gradlew")
+            copyTemplateFile(basePath, "gradlew.bat")
+            copyTemplateFile(basePath, "build.gradle.kts") { outFile ->
+                outFile.writeText(
+                    outFile
+                        .readText()
+                        .replace("KOTLIN_VERSION", wizardContext.getUserData(kotlinVersionKey) ?: "1.4.32")
+                        .replace(
+                            "GODOT_KOTLIN_JVM_VERSION",
+                            "0.1.4-3.2.3"
+                        )
+                        .replace("ANDROID_ENABLED", wizardContext.getUserData(androidEnabledKey)?.toString() ?: "false")
+                        .replace("DX_TOOL_PATH", wizardContext.getUserData(dxToolPathKey) ?: "dx")
+                )
+            }
+            copyTemplateFile(basePath, "settings.gradle.kts") { outFile ->
+                outFile.writeText(
+                    outFile
+                        .readText()
+                        .replace("PROJECT_NAME", module.project.name)
+                )
+            }
+        } else {
+            val projectBasePath = module.project.basePath ?: return
+            File(projectBasePath, "settings.gradle.kts").also { settingsFile ->
+                if (settingsFile.exists()) {
+                    val moduleRelativePath = basePath.removePrefix(projectBasePath)
+                    val modulePath = moduleRelativePath
+                        .replace("/", ":")
+                        .replace(File.separator, ":")
+                    settingsFile.appendText("\ninclude(\"$modulePath\")")
+                }
+            }
+
+            copyTemplateFile(
+                basePath,
+                "build.gradle.kts.submodule",
+                "build.gradle.kts"
+            ) { outFile ->
+                outFile.writeText(
+                    outFile
+                        .readText()
+                        .replace("ANDROID_ENABLED", wizardContext.getUserData(androidEnabledKey)?.toString() ?: "false")
+                        .replace("DX_TOOL_PATH", wizardContext.getUserData(dxToolPathKey) ?: "dx")
+                        .let { content ->
+                            if (module.parentProjectAlreadyContainsDependency(wizardContext, "godot-library")) {
+                                content.replace(
+                                    "GODOT_KOTLIN_DEPENDENCY",
+                                    "id(\"com.utopia-rise.godot-kotlin-jvm\")"
+                                )
+                            } else {
+                                content.replace(
+                                    "GODOT_KOTLIN_DEPENDENCY",
+                                    "id(\"com.utopia-rise.godot-kotlin-jvm\") version \"0.1.4-3.2.3\""
```

Comment:
Isn't that error pron to hardcode the version here ? We might forget it when we bump the version?

Reply:
I had that once but removed it again because the build props were causing an issue in the ide plugin. Don't remember what. I'll try again